docs/internals/contributing/committing-code.txt

```diff
@@ -109,8 +109,8 @@ Committing guidelines
 In addition, please follow the following guidelines when committing code to
 Django's Git repository:
 
-* Never change the published history of django/django branches! **Never
-  force-push your changes to django/django.** If you absolutely must (for
+* Never change the published history of `django/django` branches! **Never
+  force-push your changes to `django/django`.** If you absolutely must (for
```

Comment:
Just noticed that the `**` on this line are not making any difference. Is there any reason for having them here that I missed?